index.html

```diff
@@ -213,49 +191,45 @@ <h2>
           <p>
             Requests that the pointer be locked to a DOM element
             <dfn>target</dfn>. The <a>user agent</a> determines if pointer lock
-            state will be entered and upon lock state change or error must send
+            state will be entered, and upon lock state change or error MUST send
             either a <a>pointerlockchange</a> or <a>pointerlockerror</a> event
             respectively.
           </p>
           <p>
-            Pointer lock must not succeed if the document object's
-            [=Document/active sandboxing flag set=] has the <a>sandboxed
-            pointer lock browsing context flag</a> set.
+            Pointer lock MUST fail if the document object's [=Document/active
+            sandboxing flag set=] has the <a>sandboxed pointer lock browsing
+            context flag</a> set.
           </p>
           <p>
-            Pointer lock must not succeed unless the <a>target</a>'s
+            Pointer lock MUST fail unless the <a>target</a>'s
             <a>shadow-including root</a> is the <a>active document</a> of a
             [=Document/browsing context=] which is (or has an <a>ancestor
             browsing context</a> which is) in focus by a window which is in
             focus by the operating system's window manager. The <a>target</a>
             element and its [=Document/browsing context=] need not be in focus.
           </p>
           <p>
-            If a user has exited pointer lock via the <a>default unlock
-            gesture</a>, or pointer lock has not previously been entered for
-            this document, an event generated as a result of an <a>engagement
-            gesture</a> must be received by the document before
-            {{requestPointerLock()}} will succeed.
+            Pointer lock is a <a>transient activation-gated API</a>, therefore a
+            {{requestPointerLock()}} call MUST fail if <a>transient
+            activation</a> is not available.  This prevents locking upon initial
```

Comment:
Maybe be more specific, i.e. say that it must fail if this's relevant global object does not have transient activation?

Reply:
Done.